src/version.js

```diff
@@ -1 +1 @@
-export default '1.1.1';
+export default '1.1.2';
```

Comment:
You should revert this and instead bump the version in `package.json` instead. Also, the next version is `1.1.5` (see: https://npm.altislabs.com/)

Reply:
84ff277c0dca16c3461efe37cc825caddc5ada00